HangPerson/HangPerson/main.m

```diff
@@ -12,7 +12,61 @@ int main(int argc, const char * argv[]) {
     @autoreleasepool {
         
         // code goes here...
-
+        char word[5] = "pasta";
+        char dashes[5] = "_____";
+        char input;
+        char temp;
+        int guesses = 7; //Maximum tries.
+        int i;
+        int j;
+        int swaps = 5;
+        
+        
+        printf("Lets play a Hangman.\n");
+        printf("I'm thinking of a word. Can you guess it?\n");
+        
+        for(i = 0; i < guesses; i++ ) {
+            
+            printf("select a letter: \n");
+            
+            printf("%c %c %c %c %c", dashes[0], dashes[1], dashes[2], dashes[3], dashes[4]);
```

Comment:
This is very specific to a 5 character word. It would be better to loop from 0 to sizeof(dashes) and print out each letter individually.